### PR TITLE
chore: cherry-pick 2ed58f4 from chromium

### PR DESCRIPTION
**Pull request by RaisinTen**
#### Description of Change

---

Subject: [PATCH] Remove incorrect width/height adjustments

In late 2016 a change which fixed some problems around window sizing
when attaching or detaching additional displays was landed, which fixed
some genuine bugs. Unfortunately it included a subtraction of 1 from the
width and height of the Chrome window. I couldn't find any discussion of
this size adjustment and I think that it was just a misunderstanding of
how window rectangles work (inclusive versus exclusive extents).

This size adjustment causes non-maximized Chrome windows to shrink every
time a monitor is added or removed. The problematic commit was found
by the bug-filer through a bisect of more than four years of Chrome
history - I'm just landing the fix that they suggested.

Bug: 1300415
Change-Id: Ief124f584a91aa9cc3f10704b0cc1e83356dea5b
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3492658
Reviewed-by: Allen Bauer <kylixrd@chromium.org>
Commit-Queue: Bruce Dawson <brucedawson@chromium.org>
Cr-Commit-Position: refs/heads/main@{#975872}

---

Refs: https://chromium-review.googlesource.com/c/chromium/src/+/3492658
Fixes: https://github.com/electron/electron/issues/33049
Signed-off-by: Darshan Sen <raisinten@gmail.com>

#### Release Notes

Notes: Fixes an issue where adding/removing display changes the BrowserWindow size.

### CODE REVIEW

**Review by ckerr — APPROVED** [review]
LGTM. This has one upstream approval already so we may wind up backing this out of main after a future roll, but landing in main here lets trop do its thing

**Comment by dsanders11** [comment]
@ckerr, it's been merged upstream, so yea, it will come out in a future roll. For context, I was the upstream issue reporter and ran it down. I also liberally applied the target labels here, so feel free to trim them back, but it feels low risk and isolated to backport. 🙂

**Pull request merged**
